<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stitching</class>
 <widget class="QWidget" name="Stitching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color: rgb(184,191,200);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="viewLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(184,191,200);
QLabel::setAlignment (Qt::AlignVCenter );
QLabel::setAlignment (Qt::AlignHCenter );
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="rightVerticalLayout">
       <item>
        <layout class="QVBoxLayout" name="rightVBoxLayout">
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <spacer name="topVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="inputVerticalLayout">
           <item>
            <layout class="QVBoxLayout" name="pathVerticalLayout">
             <item>
              <widget class="QLabel" name="pathLabel">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
color: rgb(184,191,200);
QLabel::setAlignment (Qt::AlignVCenter );
QLabel::setAlignment (Qt::AlignHCenter );
}</string>
               </property>
               <property name="text">
                <string>File Path</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="pathHorizontalLayout">
               <item>
                <widget class="QLineEdit" name="filePathLineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="filePathPushButton">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
border-image:url(:/prefix1/Image/Path_PushButton_Icon.png)
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="panoRadioButton">
             <property name="styleSheet">
              <string notr="true">.checks {
    position: relative;
    height: 20px;
    padding: 30px;
    line-height: 20px;
}
.checks label {
    font-size: 16px;
    color: #666;
    margin-right: 10px;
    vertical-align: middle;
}
            
.checks input[type=&quot;radio&quot;] {
    position: absolute;
    width: 1px;
    height: 1px;
    padding: 0;
    margin: -1px;
    overflow: hidden;
    clip: rect(0, 0, 0, 0);
    border: 0;
}
.checks input[type=&quot;radio&quot;]+label {
    display: inline-block;
    position: relative;
    padding-left: 25px;
    cursor: pointer;
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
}
            
.checks input[type=&quot;radio&quot;]+label:before {
    content: '';
    position: absolute;
    left: 4px;
    top: 0px;
    width: 14px;
    height: 14px;
    text-align: center;
    background: #fff;
    border: 1px solid #cacece;
    border-radius: 100%;
    box-shadow: none;
}
.checks input[type=&quot;radio&quot;]:checked+label:before {
    background: #fff;
    border-color: #e86138;
} 
            
.checks input[type=&quot;radio&quot;]:checked+label:after {
    content: '';
    position: absolute;
    top: 1px;
    left: 5px;
    width: 14px;
    height: 14px;
    background: #e86138;
    border-radius: 100%;
    box-shadow: none;
}</string>
             </property>
             <property name="text">
              <string>Panorama</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cephRadioButton">
             <property name="text">
              <string>Cephalo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="middleVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>68</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonHorizontalLayout">
           <item>
            <widget class="QPushButton" name="stitchingPushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color: rgb(184,191,200);
background-color: rgb(10, 21, 49);
border: 1px solid rgb(184,191,200);
border-radius: 3px;
}
QPushButton:hover
{
border: 2px solid red;
background-color: rgb(103, 104, 114);
}</string>
             </property>
             <property name="text">
              <string>STITCHING</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearPushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color: rgb(184,191,200);
background-color: rgb(10, 21, 49);
border: 1px solid rgb(184,191,200);
border-radius: 3px;
}
QPushButton:hover
{
border: 2px solid red;
background-color: rgb(103, 104, 114);
}</string>
             </property>
             <property name="text">
              <string>CLEAR</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="bottomVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resultHorizontalLayout">
         <item>
          <spacer name="resultHorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>468</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="resultLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(230, 230, 230,150)</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reultPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgb(245, 245, 245)</string>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
